src/Idris/AbsSyntax.hs

```diff
@@ -434,7 +434,7 @@ pop_estack = do i <- getIState
     where ptail [] = []
           ptail (x : xs) = xs
 
--- | Add a class instance function.
+-- | Add an interface instance function.
```

Comment:
right, we now use `implementation` keywords to implement interface. i'll check this out.